src/main/resources/templates/email/reciept.hbs

```diff
@@ -0,0 +1,47 @@
+<html>
+
+<head>
+    <title>reciept</title>
```

Comment:
After quick research this appears to have no or very inconsistent effect, remove it.

Reply:
Was removed